src/lib/drivers/extension.js

```diff
@@ -145,7 +145,7 @@ class ExtensionDriver extends Driver {
     return new Promise((resolve, reject) => {
       chrome.tabs.query(currentTab, tabs => {
         if (chrome.runtime.lastError) {
-          return reject(chrome.runtime.lastError.message);
+          return reject(chrome.runtime.lastError);
```

Comment:
Also, every other place except this one(in the same file) was `reject(chrome.runtime.lastError)`. And I think it's good to reject ErrorType instead of String. 

Maybe - https://github.com/boopathi/lighthouse/commit/24969a74d4e409670fb434dc449ca58841a6f11a